### DISCUSSION TITLE
Own repos recommended

### DISCUSSION THREAD

**Discussion by jpnurmi** [community/community#13747] [Other Feature Feedback, Questions, & Ideas] [answered]
Suggestion: exclude users' own repos from recommendations. :slightly_smiling_face: 

![image](https://user-images.githubusercontent.com/140617/160552906-e66db9cf-a6fd-49d5-bc5b-3474485b6597.png)

**Comment by jfullstackdev** [comment, marked as answer]
yup, that's a known issue and they are already working on it

anyways, 

there is a new update on the For You feed, you can now filter it, and for those who don't want it can simply disable it by unchecking all filters

![image](https://user-images.githubusercontent.com/47092464/182012117-0b19be9d-e849-462e-909e-d20d28915c1d.png)
